x-pack/legacy/plugins/apm/public/hooks/useServiceTransactionTypes.tsx

```diff
@@ -4,17 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { loadServiceTransactionTypes } from '../services/rest/apm/services';
 import { IUrlParams } from '../context/UrlParamsContext/types';
 import { useFetcher } from './useFetcher';
+import { callApmApi } from '../services/rest/callApi';
 
 export function useServiceTransactionTypes(urlParams: IUrlParams) {
   const { serviceName, start, end } = urlParams;
-  const { data: transactionTypes = [] } = useFetcher(() => {
+  const { data = { transactionTypes: [] as string[] } } = useFetcher(() => {
```

Comment:
`as string[]` shouldn't be necessary.

```suggestion
  const { data = { transactionTypes: [] } } = useFetcher(() => {
```

Extracting the default value might also improve readability:
```js
  const INITIAL_DATA = { transactionTypes: [] };
  const { data = INITIAL_DATA } = useFetcher(() => {
```

Reply:
changed, looks better indeed 👍